cmd/juju/commands/main_test.go

```diff
@@ -288,15 +288,12 @@ var commandNames = []string{
 	"add-storage",
 	"add-unit",
 	"add-user",
-	"agree",
```

Comment:
So you removed commands around wallets and budgets, which I agree with. But you also removed commands around 'agreements' and 'agree'. Which *isn't* something that is currently going away. (though I don't think it is heavily utilized, I don't think we want to remove it at this point).
Agreements are about software licenses, not about pricing and budgets.
(I agree to the EULA from Oracle about running their JVM)

Is it possible to decouple those things? I realize both are under 'romulus' because that is the group that implemented it. Is it feasible to integrate agreements without the romulus dependency?

Reply:
I was not aware of that. I will restore the `agree` commands.

It turns out they don't actually import any romulus things as dependencies, so I'm going to move to a new dir